spk/cops/src/conf/resource

```diff
@@ -1,57 +1,55 @@
 {
-  "webservice": {
-    "migrate": {
-      "root": [{
-        "new": "cops",
-        "old": "cops"
-      }]
-    },
-    "pkg_dir_prepare": [{
-      "group": "http",
-      "mode": "0755",
-      "source": "/var/packages/cops/target/share/cops",
-      "target": "cops",
-      "user": "sc-cops"
-      }],
-    "portals": [
-      {
-        "alias": "cops",
-        "app": "com.synocommunity.packages.cops",
-        "name": "COPS",
-        "service": "cops",
-        "type": "alias"
-      }
-    ],
-    "services": [{
-      "backend": 2,
-      "display_name": "COPS",
-      "icon": "app/images/cops-{0}.png",
-      "php": {
-        "backend": 8,
-        "extensions": [
-          "gd",
-          "pdo_sqlite",
-          "sqlite3",
-          "intl",
-          "zlib"
-        ],
-        "group": "http",
-        "profile_desc": "PHP profile used by cops",
-        "profile_name": "cops",
-        "user": "sc-cops"
-      },
-      "root": "cops",
-      "service": "cops",
-      "type": "apache_php"
-    }]
-  },
-  "data-share": {
-    "shares": [{
-      "name": "{{wizard_calibre_share}}",
-      "permission": {
-        "rw": ["sc-cops"]
-      },
-      "once": true
-    }]
-  }
+	"data-share": {
```

Comment:
There doesn't seem to be any change on this file, right ? (Seems to be only formatting from what I see, just want to make sure)

Reply:
Yes, this is formatting to be consistent with the other file.